### PR TITLE
firmware: update x86 microcode

### PR DESCRIPTION
**Pull request by nasbdh9**
intel-microcode: update to 20200616
amd64-microcode: update to 20191218

Run test: x86_64

Signed-off-by: Tan Zien <nabsdh9@gmail.com>

### CODE REVIEW

File: package/firmware/amd64-microcode/Makefile [hunk 1]
```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=amd64-microcode
-PKG_VERSION:=20180524
+PKG_VERSION:=20191218
```
Comment (hauke) [inline package/firmware/amd64-microcode/Makefile:11]: Henrique de Moraes Holschuh from Debian warned us to take version 20191021, see here:
https://www.mail-archive.com/openwrt-devel@lists.openwrt.org/msg49810.html

I do not know if the problem he is talking about is fixed now.

**Review by hauke — COMMENTED** [review]
On which system did you test this?

File: package/firmware/amd64-microcode/Makefile (hunk 1 above)
Comment (hauke) [reply package/firmware/amd64-microcode/Makefile:11]: It looks like 20191218 removed one microcode update which caused problems:
https://metadata.ftp-master.debian.org/changelogs//non-free/a/amd64-microcode/amd64-microcode_3.20191218.1_changelog
I think this is ok.

Comment (nasbdh9) [reply package/firmware/amd64-microcode/Makefile:11]: Yes, the microcode of zen2 processor structure is not included in 20191218

**Comment by hauke** [comment]
Thank you for the pull request, it was applied to master.

**Comment by sch-m** [comment]
@hauke Can you cherry-pick these commits to openwrt-19.07 as well, since some CVEs have been fixed.

**Pull request closed without merging**